modules/microshift-nw-router-con.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-nw-router.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-about-router-config_{context}"]
+= About configuring the router
+
+The default {microshift-short} ingress router is always on, running on all IP addresses on fixed ports. The default router settings also allow access to any namespace. Some applications running on top of {microshift-short} might not even need the default router and instead create their own. To make ingress optional, you can configure the router settings to manage which ports, if any, are exposed to network traffic.
```

Comment:
The router is only on if it is configured to do so, now you can disable it by using `Removed` in the `ingress.Status`.
The second statement about the ports is also impacted by this.